app/stylesheets/boilerplate/buttons.scss

```diff
@@ -0,0 +1,19 @@
+@import '../config/type';
+@import '../config/colours';
+@import '../config/dimensions';
+@import '../config/box';
+
+
+a.button-link {
+  @include border-radius(3px);
+  background: $theme-blue;
+  color: $text-inverse;
+  display: inline-block;
+  font-weight: $title-weight;
```

Comment:
Lets add a `$button-weight` variable for this, it can even be `$button-weight: $title-weight`, but just so they are not tightly linked.